corehq/elastic.py

```diff
@@ -69,7 +69,7 @@ def get_es_export():
         retry_on_timeout=True,
         max_retries=3,
         # Timeout in seconds for an elasticsearch query
-        timeout=30,
+        timeout=300,
```

Comment:
Just curious why we set this? if we can just keep increasing it why not keep it as a very high number?
We want to keep some value to that we can catch very very big searches.

Reply:
Agree on that. This might be too far, but i think its reasonable for exports and other async processes to have a much higher timeout.

Reply:
30 vs 10 wasnt super significant